app/views/application/_flash.html.erb

```diff
@@ -1,11 +1,15 @@
-<%= content_tag :div, id: 'flash-messages' do %>
-  <% flash.each do |key, value| %>
-    <%= content_tag :div, class: "alert alert-dismissible #{flash_key_to_bootstrap_class(key)}" do %>
-      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-        <span aria-hidden="true">&times;</span>
-      </button>
+<%= content_tag :div, class: 'container' do %>
+  <%= content_tag :div, class: 'row' do %>
+    <%= content_tag :div, class: 'col-sm-12 col-md-6 col-lg-6 col-md-offset-3 col-lg-offset-3 flash-message' do %>
```

Comment:
you can change `contect_tag` to just `<div>` here, as they don't require ruby code :)